lib/resources/r4/charge_item.yaml

```diff
@@ -105,3 +105,24 @@ fields:
         - display: Bill Code Schedule
           system: https://fhir.cerner.com/&lt;EHR source id&gt;/codeSet/14002
           info_link: https://fhir.cerner.com/millennium/r4/proprietary-codes-and-systems/#code-set-14002-bill-code-schedule
+
+  - name: performer
+    type: List of BackboneElement
+    description: Who performed charged service.
+    action: terminology
+    url: https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.performer
+    children:
+
+      - name: function
+        type: CodeableConcept
+        description: Distinguishes the type of involvement of the performer in the event.
+        action: terminology
+        binding:
+          description: A code that identifies the function of the performer.
+          terminology:
+            - display: Performer function codes
+              system: http://terminology.hl7.org/CodeSystem/v3-ParticipationType
+              info_link: http://hl7.org/fhir/R4/valueset-performer-function.html
+            - display: Participation codes
+              system: http://terminology.hl7.org/CodeSystem/v2-0912
+              info_link: http://terminology.hl7.org/CodeSystem/v2-0912
```

Comment:
can you change the info_links to use https instead of http?

Reply:
updated system and info_links to https here https://github.com/cerner/fhir.cerner.com/pull/682/commits/144f92e439aba8231f1986ae1675c80928711b36#diff-7ab995fcc2135db120dc94b545fa48898edd21d9d220eee5efae95920c369dcbR124-R128

Reply:
revert system links back to http here https://github.com/cerner/fhir.cerner.com/pull/682/commits/1df8b1f2f90cb56f78df6812be0f33692283d5ae#diff-7ab995fcc2135db120dc94b545fa48898edd21d9d220eee5efae95920c369dcbR124-R127